Origin: uos
Last-Update: 2022-05-23

--- jimtcl-0.79+dfsg0.orig/autosetup/autosetup-config.guess
+++ jimtcl-0.79+dfsg0/autosetup/autosetup-config.guess
@@ -1039,6 +1039,9 @@ EOF
     sparc:Linux:*:* | sparc64:Linux:*:*)
 	echo "$UNAME_MACHINE"-unknown-linux-"$LIBC"
 	exit ;;
+    sw_64:Linux:*:*)
+        echo "$UNAME_MACHINE"-unknown-linux-"$LIBC"
+        exit ;;
     tile*:Linux:*:*)
 	echo "$UNAME_MACHINE"-unknown-linux-"$LIBC"
 	exit ;;
--- jimtcl-0.79+dfsg0.orig/autosetup/autosetup-config.sub
+++ jimtcl-0.79+dfsg0/autosetup/autosetup-config.sub
@@ -308,6 +308,7 @@ case $basic_machine in
 	| sparc | sparc64 | sparc64b | sparc64v | sparc86x | sparclet | sparclite \
 	| sparcv8 | sparcv9 | sparcv9b | sparcv9v \
 	| spu \
+	| sw_64 \
 	| tahoe | tic4x | tic54x | tic55x | tic6x | tic80 | tron \
 	| ubicom32 \
 	| v850 | v850e | v850e1 | v850e2 | v850es | v850e2v3 \
